pontoon/base/urls.py

```diff
@@ -48,6 +48,11 @@
         views.download_translation_memory,
         name='pontoon.download_tmx'),
 
+    # Locale-agnostic links
+    url(r'^projects/(?P<slug>[\w-]+)/(?P<part>.+)/$',
+        views.locale_agnostic,
```

Comment:
I suggest we changed the view name from `locale_agnostic` to `translate_locale_agnostic`. You'll also need to change it in the views.py.